app/helpers/startup_helper.rb

```diff
@@ -0,0 +1,2 @@
+module StartupHelper
```

Comment:
Why do you need these empty files?

Reply:
should i remove them?

Reply:
yeah remove them and their spec files while we arent using them, we can add them later if they are needed.